teacher.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name)
+    super(age, name)
+    @specialization = specialization
+  end
+
+  def can_use_services
+    true
+  end
```

Comment:
- [ ] This method should override the one in the Person class. That means we should add "?" to it as previously required.

  ```
  def can_use_services?
    true
  end
  ```